shared/dataflow/codeql/dataflow/internal/DataFlowImplCommon.qll

```diff
@@ -778,9 +780,13 @@ module MakeImplCommon<InputSig Lang> {
        * `ctx`.
```

Comment:
This qldoc needs an update - it's no longer correct now that you've shuffled the columns around.

Reply:
I thought it was still correct; what would you suggest updating it to?

Reply:
The `call` is no longer in the context `ctx`. It used to be that the return chain was from `result` to `call` and then further to `ctx`, so the return context was the `result`-to-`call` edge. Now that edge is the `CallContextReturn ctx`, the `result` is the callable in which `ctx` makes sense, and `call` is where we can return to (the prior `ctx`).
So keeping the `DataFlowCallable` as the result is going to make the qldoc phrasing awkward, since that's not really the "result" - rather, it is the `call` that's the "result". I.e. "Holds if `result`-to-`call` is a viable return edge in the context `ctx`."